perun-db/oracle.sql

```diff
@@ -1206,7 +1193,7 @@ create index IDX_FK_USREX_USERSRC on user_ext_sources(ext_sources_id);
 create index IDX_FK_MEM_USER on members(user_id);
 create index IDX_FK_MEM_VO on members(vo_id);
 create index IDX_FK_HOST_FAC on hosts(facility_id);
-create index IDX_FK_EXSRV_SRV on exec_services(service_id);
+create index IDX_FK_SRV_SRV on services(service_id);
```

Comment:
Should be ``create index IDX_FK_SRV_SRV on services(id);``

Reply:
I also noticed, that this index is not part of the both changelogs or definition for postgres. We should investigate and align all db types.